src/main/java/Task.java

```diff
@@ -0,0 +1,31 @@
+public class Task {
+
+    protected String content;
+    protected boolean isDone;
+    private final String TASK_SYMBOL = "[ ]";
+
+    public Task(String content) {
+        this.content = content;
+        this.isDone = false;
+    }
+
+    public String getTask() {
```

Comment:
```suggestion
    public String getDescription() {
```
This would be clearer.